code/modules/mob/living/carbon/human/human.dm

```diff
@@ -18,7 +18,7 @@
 		dna = new /datum/dna(null)
 		// Species name is handled by set_species()
 
-	set_species(new_species, 1, delay_icon_update = 1, skip_same_check = TRUE)
+	set_species(new_species, 1, delay_icon_update = 1, skip_same_check = TRUE, new_mob = TRUE)

```

Comment:
Maintains legacy behaviour of species organ generation for character creation. Everything is made from scratch